### PR TITLE
NET 6409

### PR DESCRIPTION
**Pull request by absolutelightning**
### Description

Added docs for Consul K8s Proxy Stats Command

### PR Checklist

* [x] updated test coverage
* [x] external facing docs updated
* [x] appropriate backport labels added
* [x] not a security concern

### CODE REVIEW

File: website/content/docs/k8s/k8s-cli.mdx [hunk 1]
```diff
@@ -781,6 +782,86 @@ file                            info
 kafka                           info
 lua                             info
 ```
+
+### `proxy stats`
+
+The `proxy stats` command allows you to inspect the cluster stats for Envoy proxies running on a given Pod.
```
Comment (david-yu) [inline website/content/docs/k8s/k8s-cli.mdx:788]:
```suggestion
The `proxy stats` command allows you to inspect the Envoy cluster stats for Envoy proxies running on a given Pod.
```

File: website/content/docs/k8s/k8s-cli.mdx [hunk 2]
```diff
@@ -781,6 +782,86 @@ file                            info
...
+| Flag                                 | Description                                           | Default                                                                                                                |
+| ------------------------------------ | ----------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------- |
+| <nobr>`-namespace`, `-n`</nobr>      | `String` The Kubernetes namespace to list proxies in. | Current [kubeconfig](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) namespace. |
+
+Refer to the [Global Options](#global-options) for additional options that you can use
+when installing Consul on Kubernetes.
+
+#### Example Commands
+
+Display cluster stats of Envoy in a given pod in default namespace.
```
Comment (david-yu) [inline website/content/docs/k8s/k8s-cli.mdx:802]:
```suggestion
Display the Envoy cluster stats in a given pod in default namespace.
```

**Comment by david-yu** [comment]
Could you also add the command to the Command TOC here? https://consul-7g86tczfl-hashicorp.vercel.app/consul/docs/k8s/k8s-cli#commands

**Comment by david-yu** [comment]
We should also backport to 1.17.x, 1.16.x and 1.15.x

File: website/content/docs/k8s/k8s-cli.mdx [hunk 3]
```diff
@@ -781,6 +783,86 @@ file                            info
...
+cluster.consul-dataplane.http2.deferred_stream_close: 0
+cluster.consul-dataplane.http2.dropped_headers_with_underscores: 0
+cluster.consul-dataplane.http2.header_overflow: 0
+cluster.consul-dataplane.http2.headers_cb_no_stream: 0
+cluster.consul-dataplane.http2.inbound_empty_frames_flood: 0
+cluster.consul-dataplane.http2.inbound_priority_frames_flood: 0
+.........
+```
+
+Display cluster stats of Envoy in a given pod in different namespace.
```
Comment (david-yu) [inline website/content/docs/k8s/k8s-cli.mdx:838]:
```suggestion
Display the Envoy cluster stats in a given pod in different namespace.
```

**Review by david-yu — APPROVED** [review]

**Comment by david-yu** [comment]
Looks good, we should wait to merge until the next patch release, which should include this in consul-k8s 1.3.x, 1.2.x, and 1.1.x

**Comment by absolutelightning** [comment]
> Looks good, we should wait to merge until the next patch release, which should include this in consul-k8s 1.3.x, 1.2.x, and 1.1.x

Please clarify till when we need to wait?

**Comment by david-yu** [comment]
This would roughly be 5 weeks from now for the next set of patch releases.

**Comment by david-yu** [comment]
@absolutelightning Could you resolve the failed docs backports?

**Pull request merged**